py-polars/tests/unit/io/test_csv.py

```diff
@@ -584,6 +584,10 @@ def test_csv_quote_char() -> None:
         assert out.shape == (9, 3)
         assert_frame_equal(out, expected)
 
+    # non-standard quote char
+    df = pl.DataFrame({"x": ["", "0*0", "xyz"]})
+    assert df.write_csv(quote="*") == "x\n**\n*0**0*\nxyz\n"
```

Comment:
Why is the asterisk duplicated here?
```suggestion
    assert df.write_csv(quote="*") == "x\n**\n*0*0*\nxyz\n"
```

Reply:
Okay, thanks for the explanation!